third_party/docfx/templates/devsite/partials/uref/class.tmpl.partial

```diff
@@ -45,7 +45,8 @@
 {{/parameters.0}}
 {{#parameters}}
     <tr>
-      <td>{{{type.specName.0.value}}}</td>
+      {{#var_type}}<td>{{{var_type}}}</td>{{/var_type}}
+      {{^var_type}}<td>{{{type.specName.0.value}}}</td>{{/var_type}}

```

Comment:
Perhaps swap this so the `<td>` is always there and the template is inside. That way, we avoid the extra newline.

Reply:
Do you mean like this instead?
```
      <td>
        {{#var_type}}{{{var_type}}}{{/var_type}}
        {{^var_type}}{{{type.specName.0.value}}}{{/var_type}}
      </td>
```
Or have them squished to one line like
```
      <td>{{#var_type}}{{{var_type}}}{{/var_type}}{{^var_type}}{{{type.specName.0.value}}}{{/var_type}}</td>
```

Reply:
One line, if it's not too unreadable.